src/main/scala/com/campudus/tableaux/database/model/TableauxModel.scala

```diff
@@ -69,15 +69,13 @@ sealed trait StructureDelegateModel extends DatabaseQuery {
   }
 }
 
-class TableauxModel(override protected[this] val connection: DatabaseConnection) extends DatabaseQuery with StructureDelegateModel {
+class TableauxModel(
+  override protected[this] val connection: DatabaseConnection,
+  override protected[this] val structureModel: StructureModel
```

Comment:
Is this to prevent creating multiple `StructureModel` instances in `TableauxModel` and `RouterRegistry`? Did this cause trouble? Good catch :)